plugins/modules/win_dns_zone.py

```diff
@@ -80,6 +80,13 @@
       - At least one server is required.
     elements: str
     type: list
+  computer_name:
+    description:
+      - Specifies a DNS server.
+      - You can specify an IP address or any value that resolves to an IP
+        address, such as a fully qualified domain name (FQDN), host name, or
+        NETBIOS name.
+    type: str
```

Comment:
This is a new option, it should have `version_added: 1.2.0` to tell users when this was first available in the docs.